Disconnect, Shutdown, and Restart Message Replies.

ADetectionAlgorithm opened the pull request:
Bot posts back to the server when the {}Disconnect, {}Shutdown, or {}Restart message is fired successfully.  This shows the operator that the bot did in fact recognize the command, something that isn't always apparent when not looking at console, especially if the bot needs to restart.

Commit:
Added restart, shutdown, disconnect message replies

Bot posts to server when disconnecting, restarting, and shutting down,
as a way to show the owner that the command did in fact work.

imayhaveborkedit commented on musicbot/bot.py:
```diff
@@ -1781,14 +1781,32 @@ def check(message):
+            message.channel,
+            'Disconnecting from %s...' % server,
+            expire_in=30,
+            also_delete=message if self.config.delete_invoking else None
```
`expire_in` and `also_delete` are pointless since the bot will not maintain the state of the messages to delete them after restarting.

ADetectionAlgorithm replied:
Yeah, it works on the disconnect cmd, but not on the restart or shutdown cmds.  I originally left them because I didn't know if they would work without them, but after testing, they have been removed.

Commit:
Fixed Commands

Accidentally used server instead of voice channel on disconnect, missed
message as parameter on restart and shutdown.

Commit:
Removed unnecessary parameters

removed unnecessary parameters in shutdown and restart commands

Commit:
Fixed Disconnect Command

only expires if config is set for botmessages to expire, honors config.

Commit:
Attempt #2

Commit:
Revert "Attempt #2"

This reverts commit 81428bc01c892047bcfb784d0d8fe8669ef4c27b.
